config/iroha_test_config.toml

```diff
@@ -0,0 +1,35 @@
+[iroha]
+chain_id = "00000000-0000-0000-0000-000000000000"
+public_key = "ed01201C61FAF8FE94E253B93114240394F79A607B7FA55F9E5A41EBEC74B88055768B"
+private_key.digest_function = "ed25519"
+private_key.payload = "282ED9F3CF92811C3818DBC4AE594ED59DC1A2F78E4241E31924E101D6B1FB831C61FAF8FE94E253B93114240394F79A607B7FA55F9E5A41EBEC74B88055768B"
```

Comment:
Having the key split across two different fields is indeed unfortunate =(

Reply:
Can we use multihash for private keys too?

Reply:
Maybe we can use [Inline Table](https://toml.io/en/v1.0.0#inline-table)?

Reply:
We can, but it is really a matter of style. All of these are equivalent:

```toml
private_key.digest_function = "xxx"
private_key.payload = "yyy"
```

```toml
private_key = { digest_function = "xxx", payload = "yyy" }
```

```toml
[private_key]
digest_function = "xxx"
payload = "yyy"
```

I didn't use inline table because of the very long payload hex, making it less readable.

Reply:
> We can, but it is really a matter of style. All of these are equivalent:
> 
> ```toml
> private_key.digest_function = "xxx"
> private_key.payload = "yyy"
> ```
> 
> ```toml
> private_key = { digest_function = "xxx", payload = "yyy" }
> ```
> 
> ```toml
> [private_key]
> digest_function = "xxx"
> payload = "yyy"
> ```
> 
> I didn't use inline table because of the very long payload hex, making it less readable.

I would be more comfortable seeing 2nd or 3rd format

Reply:
Used inline table.